### DISCUSSION TITLE
Help on finding ppyolov2_r50vd_dcn_365e_publaynet model config file

### DISCUSSION THREAD

**Discussion by WYY220062** [PaddlePaddle/PaddleOCR#13915] [Q&A]
Hi!
I would like to run the inference of model ppyolov2_r50vd_dcn_365e_publaynet
Find the inference model here https://github.com/PaddlePaddle/PaddleOCR/blob/main/ppstructure/docs/models_list.md.
Can you point me the config file and corresponding sample code?
Thank you in advance!

**Comment by GreatV** [comment]
You can try to modify the configuration file based on this document.
https://github.com/PaddlePaddle/PaddleDetection/tree/release/2.5/configs/picodet/legacy_model/application/layout_analysis
